BookingsApi/BookingsApi.DAL/Queries/GetJusticeUserListQuery.cs

```diff
@@ -24,7 +27,23 @@ public GetJusticeUserListQueryHandler(BookingsDbContext context)
 
         public async Task<List<JusticeUser>> Handle(GetJusticeUserListQuery query)
         {
-            return await _context.JusticeUsers.Include(x => x.UserRole).ToListAsync();
+            string term = query.Term;
+            if (string.IsNullOrEmpty(term))
+            {
+                return await _context.JusticeUsers.Include(x => x.UserRole).ToListAsync();
+            }
+            else
+            {
+                return await _context.JusticeUsers
```

Comment:
You do not need to await on tasks here. it would be more efficient to have the caller await on the task

Reply:
I agree but for this we need to refactor all the queries. We should not change only one query but all of them to be consistent. Happy to open a refactor ticket for this.